lib/index.js

```diff
@@ -36,13 +35,22 @@ var method = module.exports = (obj, key, method) => {
   }
   flag.add(key);
 
-  Object.defineProperty(obj, key, {
-    writable: true,
+  var descriptor = {
     configurable: true,
     enumerable: true,
-    value: method
-  });
+  };
+  
+  if (value && (value.get || value.set)) {
```

Comment:
it's confused that is a descriptor or an object with get/set method

Reply:
yes, I think should keep the old method mock original. And and new api  muk.getter、muk.setter with prop

Reply:
An object with get/set method? But that's what a descriptor looks like